docs/docs/overview/quickstart/get-started-with-python.md

```diff
@@ -0,0 +1,198 @@
+
+A notification workflow is the blueprint for the notifications that will be sent. It holds the entire flow of messages sent to the subscriber. This is where all the different channels are tied together under a single entity.
+
+The workflow includes the following:
+
+- Notification workflow name and Identifier
+- Channel tailored content:
+
+| Channel | Content Style                                                                                 |
+| ------- | --------------------------------------------------------------------------------------------- |
+| Email   | 1. Custom Code (HTML) with option to use custom variables via the handlebars , {{ }}, syntax. |
+|         | 2. Click and place UI items with the visual template editor.                                  |
+| SMS     | Text with the option to use handlebars syntax, {{ }} to inject custom variables.              |
+| Chat    | Text with the option to use handlebars syntax, {{ }} to inject custom variables.              |
+| In-App  | Text                                                                                          |
+
+Note: Proper authorization needs to be set for the Chat channel for subscribers.
+
+Please proceed to create a notification workflow.
+
+1. Click “Workflows” on the left sidebar of your Novu dashboard.
+2. Click the “Create Workflow” button on the top right.
+3. The name of a new workflow is currently "Untitled." Rename it to a more suitable title.
+4. Select "Email" as the channel you want to add.
+   ![set-email.png] (https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/set-email_wavtrn.png)
+
+5. Click on the recently added channel, fill the email subject and click “Update”.
+   ![update_email_template.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/update_email_template_ivn0jv.png)
+
+6. Click on the “Test” tab and send a test email to verify your notification workflow.
+   ![send_test_email.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/send_test_email_ngzmth.png)
+
+You should get an email within seconds. Yaaay, you have successfully sent your first notification via the Novu dashboard! Now, let’s take it a step further to trigger notifications via code.
+
+## Create A Subscriber
+
+The recipients of a triggered notification are called subscribers.
+
+Click “Subscribers” on the left sidebar of the Novu dashboard to see all subscribers. By default, the dashboard will display a subscriber, as you were added automatically during sign-up.
+
+![subscriber_id.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1688331839/Screenshot_2023-07-03_at_0.02.53_jmkhi3.png)
+
+Now, let's create a subscriber on Novu. Copy and paste the following code to do so:
+
+```python
+# Create a subscriber
+
+from novu.api.subscriber import SubscriberApi
+from novu.dto.subscriber import SubscriberDto
+   
+your_subscriber_id = "123"; # Replace this with a unique user ID that matches your database.
```

Comment:
@unicodeveloper I like the approach of @ankur0904 to extract the `subscriberID` into a separate variable (as it allows us to comment and tell the reader about it). Shall we update all the other quick start guides to use this very approach or do you feel the approach that we've been using is better and Ankur should change it to that style as well (for uniformity across guides)?

Reply:
@sumitsaurabh927 Thanks

Reply:
its fine, other guides can still remain the way it is.